### PR TITLE
LPS-155453 - Insert Placeholder in input area inside form

### PR DESCRIPTION
**Pull request by rafaelpdemelo**

### CODE REVIEW

**Comment by rafaelpdemelo** [comment]
ci:test:sf

**Comment by rafaelpdemelo** [comment]
ci:test:relevant

**Comment by rafaelpdemelo** [comment]
ci:test:relevant

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/ExpressionBuilder.tsx [hunk 1]
```diff
@@ -23,6 +23,9 @@ import {FieldBase} from './FieldBase';
 
 import './ExpressionBuilder.scss';
 
+const TEXT_PLACEHOLDER =
+	'<#-- Create the condition of the action using Expression Builder. Type "${" to use the autocomplete feature. -->';
```
Comment (danvb112) [inline modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/ExpressionBuilder.tsx:27]: Hey @rafaelpdemelo, You must create a language key for this message, to do this you can go to `Language.properties` and add the key and the formatted message, then you can run `gw buildlang` to build a copy of the message in all languages supported by Liferay Portal.

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/ExpressionBuilder.tsx [hunk 2]
```diff
@@ -170,6 +173,7 @@ export function ExpressionBuilderModal({sidebarElements}: IModalProps) {
 				<CodeEditor
 					error={error}
 					onChange={() => {}}
+					placeholder={TEXT_PLACEHOLDER}
```
Comment (danvb112) [inline modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/ExpressionBuilder.tsx:176]: @rafaelpdemelo, as you are using this placeholder only once, I don't think you need to create a const for this, you can add the string with the language key I suggested above to do this, like:

```tsx
placeholder={`<#-- {Liferay.Language.get(create-the-condition-of-the-action-using-expression-Builder-type-to-use-the-autocomplete-feature.)} -->`}

```

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @rafaelpdemelo, I have jus tow suggestions about your PR, fell free to ask me about any question. :wink:

**Comment by rafaelpdemelo** [comment]
ci:test:sf

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/ExpressionBuilder.tsx (hunk 2 above)
Comment (rafaelpdemelo) [reply modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/ExpressionBuilder.tsx]: Great @danvb112 , I'm sending new commits and go run CI.

**Comment by rafaelpdemelo** [comment]
ci:stop

**Comment by rafaelpdemelo** [comment]
ci:test:sf

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by rafaelpdemelo** [comment]
ci:forward

**Pull request closed without merging**